crypto/message.go

```diff
@@ -223,6 +223,24 @@ func (msg *PGPMessage) GetArmoredWithCustomHeaders(comment, version string) (str
 	return armor.ArmorWithTypeAndCustomHeaders(msg.Data, constants.PGPMessageHeader, version, comment)
 }
 
+// getEncryptionKeyIds Returns the key IDs of the keys to which the session key is encrypted.
+func (msg *PGPMessage) getEncryptionKeyIDs() []uint64 {
+	packets := packet.NewReader(bytes.NewReader(msg.Data))
+	var err error
+	var ids []uint64
+	for {
+		var p packet.Packet
+		if p, err = packets.Next(); err == io.EOF {
+			break
+		}
+		enc, ok := p.(*packet.EncryptedKey)
+		if ok {
+			ids = append(ids, enc.KeyId)
+		}
+	}
+	return ids
```

Comment:
Shouldn't we issue an error in case IDs is empty?

Reply:
This is matter of preference, for example KeysById in x/crypto could return empty array of keys. In javascript this function could also return empty array. I could add an error, through probably bool flag is better.